recipes-kernel/linux/linux-at91_5.4.104.bb

```diff
@@ -21,6 +21,7 @@ KBRANCH_sama7g5ek-sd = "sama7g5_early"
 KBRANCH_sama7g5ek-emmc = "sama7g5_early"
 SRC_URI = "git://github.com/linux4sam/linux-at91.git;protocol=git;branch=${KBRANCH}"
 SRC_URI += "file://defconfig file://0001-Add_HID_DTS.patch"
+SRC_URI += "file://defconfig file://0002-Commented_serial_dbg_messages.patch"
```

Comment:
Just for next time: careful you added again defconfig and would be cleaner to just append to above line.